jobs/integr8ly/pds-upgrade.yaml

```diff
@@ -0,0 +1,147 @@
+---
+
+- job:
+    name: pds-upgrade
+    project-type: pipeline
+    description: "Upgrades Integreatly on PDS cluster by using bastion server as Jenkins slave and executing Ansible upgrade playbook there."
+    sandbox: false
+    concurrent: true
+    parameters:
+        - string:
+            name: REPOSITORY
+            default: https://github.com/integr8ly/installation.git
+            description: "Repository of the Integreatly installer"
+        - string:
+            name: BRANCH
+            default: 'master'
+            description: "Branch of the installer repository"
+        - string:
+            name: YOURCITY
+            description: "City or Customer (5 char min.) plus the generated hash, e.g. qebrno-5d10 [required]"
+        - string:
+            name: ANSIBLE_USER
+            default: ec2-user
+            description: "User for Ansible to access the master node of target cluster"
+        - string:
+            name: BASTION_USER
+            default: ec2-user
+            description: "User capable of SSH-ing to bastion server"
+        - string:
+            name: BASTION_PRIVATE_KEY_ID
+            description: "ID of SSH Credentials (private key) used for SSH-ing to bastion"
+    dsl: |
+        import hudson.model.*
+        import jenkins.model.*
+        import hudson.slaves.*
+        import hudson.slaves.EnvironmentVariablesNodeProperty.Entry
+
+        import hudson.plugins.sshslaves.verifiers.*
+
+        String MASTER_URL = "master1.${YOURCITY}.internal"
+        String BASTION_URL = "bastion.${YOURCITY}.openshiftworkshop.com"    
+        String bastionLabel = "${BASTION_URL}-slave"                      
+        
+        try {
+            timeout(60) { ansiColor('gnome-terminal') { timestamps {
```

Comment:
not a big deal, but 60 minutes is probably too long for timeout. I think 20 is more than enough